pkg/testcase/upgradenodereplacement.go

```diff
@@ -27,13 +28,14 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, flags *customflag.FlagConfi
 	}
 
 	resourceName := os.Getenv("resource_name")
+	fmt.Println("resource-name: ", resourceName)
 	awsDependencies, err := aws.AddClient(cluster)
 	Expect(err).NotTo(HaveOccurred(), "error adding aws nodes: %s", err)
 
 	// create server names.
 	var serverNames, instanceServerIds, newExternalServerIps, newPrivateServerIps []string
 	for i := 0; i < len(cluster.ServerIPs); i++ {
-		serverNames = append(serverNames, fmt.Sprintf("%s-distros-qa-server-replace-%d", resourceName, i+1))
+		serverNames = append(serverNames, fmt.Sprintf("%s-%s-server-replace%d", resourceName, resourceTag, i+1))
```

Comment:
sorry we dont need `resourceTag`

I forgot to remove, since its already set on aws.
https://github.com/rancher/distros-test-framework/blob/4f65fe8b7223db7bd9bfabea68e92165e8ccc47a/pkg/aws/aws.go#L321

Reply:
I actually removed it from there and added another tag called `Team: distros-qa`... else it will be like `server-replace1-distros-qa`...

Reply:
Done